apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -33,6 +36,10 @@ class SettingsCog extends Component {
     this.open = this.open.bind(this);
     this.beforeClose = this.beforeClose.bind(this);
     this.close = this.close.bind(this);
+    this.manageAssets = this.manageAssets.bind(this);
+    this.toggleMakerToolkit = this.toggleMakerToolkit.bind(this);
+    this.confirmEnableMaker = this.confirmEnableMaker.bind(this);
+    this.hideConfirmation = this.hideConfirmation.bind(this);
```

Comment:
rather than binding all of these in the constructor, use class property syntax:

```jsx
hideConfirmation = () => {
  this.setState({confirmingEnableMaker: false});
}
```